client/src/r_sky.cpp

```diff
@@ -174,8 +174,6 @@ void R_GenerateLookup(int texnum, int *const errors); // from r_data.cpp
 
 void R_InitSkyMap()
 {
-	fixed_t fskyheight;
-
```

Comment:
This variable still needs to be declared somewhere. Looks like an earlier commit had it declared at the point it's first used down below.